src/__tests__/ReactHardwareComponent-test.js

```diff
@@ -25,55 +30,45 @@ describe('ReactHardwareComponent', () => {
     });
   });
 
-  describe('receiveComponent', function() {
-    let gen;
-    beforeEach(function() {
-      const NodeStub = function(initialProps) {
-        this._currentElement = {props: initialProps};
-        this._rootNodeID = 'test';
-      };
-      NodeStub.prototype = new ReactHardwareComponent({
-        validAttributes: {
-          pin: true,
-          value: true,
-        },
+  describe('pin', () => {
+    describe('receiveComponent', function() {
```

Comment:
nit: while you’re in here could you update these `function() {`s to `() => {`? no issue either way